### ISSUE TITLE
Minimal job environment in Admin console should not use PSScriptAnalyzer or have the option to disable it

### ISSUE THREAD

**Issue by RockoTheeHut** [ironmansoftware/powershell-universal#3130]
### Summary of the new feature / enhancement

When using a minimal job environment the PSScriptAnalyzer should be disabled, or at least an option is added to disable it. I have some developers wanting to make minor tweaks to some Python code and it will not allow saving from within the admin console. It shows a ton of errors because it isn't Powershell code and it will not allow saving because of it. 

It isn't an issue when saving code directly to the script, but it would be nice to have the option.

**Comment by adamdriscoll** [comment]
Can you let me know which editor? We have logic to avoid syntax checks in the Automation \ Scripts editor so I'm not seeing this happen.

**Comment by adamdriscoll** [comment]
If you are still experiencing this, please let me know and I can reopen.

**Comment by RockoTheeHut** [comment]
Hey, so sorry I didn't see the other response. 

It looks like we figured out the part of the issue. You can run python from a ps1 script in PSU with the way minimal environments work. Our initial script was using minimal environment, running Python, executing python code inside of a ps1file. When we renamed the script to .py files we can now save changes. 

The editor still shows we have a ton of warnings and syntax errors. The warnings appear correct, but there are no syntax errors flagged in the script and the script runs correctly. So it looks like a minor glitch with the Syntax Errors box is the thing now. 

Long story short, moving the file to use .py fixed the big issues we were seeing. 

![image](https://github.com/ironmansoftware/issues/assets/136116901/11bc92e0-54cf-4872-ba6d-2955120afeca)
